pkg/mount/mount_unix_test.go

```diff
@@ -3,21 +3,23 @@
 package mount // import "github.com/docker/docker/pkg/mount"
 
 import (
+	"fmt"
 	"os"
 	"path"
+	"strings"
 	"testing"
 )
 
 func TestMountOptionsParsing(t *testing.T) {
-	options := "noatime,ro,size=10k"
+	options := "noatime,ro,noexec,size=10k"
 
 	flag, data := parseOptions(options)
 
 	if data != "size=10k" {
 		t.Fatalf("Expected size=10 got %s", data)
 	}
 
-	expectedFlag := NOATIME | RDONLY
+	expectedFlag := NOATIME | RDONLY | NOEXEC
```

Comment:
maybe s/expectedFlag/expectedFlags/ while you're at it.

Reply:
Thank you for the detailed review. I will update the PR, sorry for the delay.